### ISSUE TITLE
[BUG] NPE when search Open Library

### ISSUE THREAD

**Issue by hoshsadiq** [mateusz-bak/openreads-android#163]
**Describe the bug**
I got a NPE when searching in Open Library to add a book.

```
type: crash
osVersion: google/raven/raven:13/TP1A.221005.002/2022100300:user/release-keys
package: software.mdev.bookstracker:28
process: software.mdev.bookstracker

java.lang.NullPointerException: Attempt to invoke interface method 'java.util.Iterator java.util.List.iterator()' on a null object reference
	at software.mdev.bookstracker.ui.bookslist.fragments.AddBookSearchFragment.onViewCreated$lambda-7(AddBookSearchFragment.kt:27)
	at software.mdev.bookstracker.ui.bookslist.fragments.AddBookSearchFragment.d(Unknown Source:0)
	at x7.b.d(Unknown Source:10)
	at androidx.lifecycle.LiveData.b(LiveData.java:30)
	at androidx.lifecycle.LiveData.c(LiveData.java:56)
	at androidx.lifecycle.LiveData.j(LiveData.java:15)
	at androidx.lifecycle.u.j(MutableLiveData.java:1)
	at androidx.lifecycle.LiveData$a.run(LiveData.java:19)
	at android.os.Handler.handleCallback(Handler.java:942)
	at android.os.Handler.dispatchMessage(Handler.java:99)
	at android.os.Looper.loopOnce(Looper.java:201)
	at android.os.Looper.loop(Looper.java:288)
	at android.app.ActivityThread.main(ActivityThread.java:7904)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:548)
	at com.android.internal.os.ExecInit.main(ExecInit.java:49)
	at com.android.internal.os.RuntimeInit.nativeFinishInit(Native Method)
	at com.android.internal.os.RuntimeInit.main(RuntimeInit.java:355)

```

**To Reproduce**
Open OpenReads
Click "For later"
`+` icon
Search in Open Library
Search for `The Mirage matt ruff`

**Expected behavior**
Should show me a list

**Screenshots**

**Additional info(please complete the following information):**
 - Manufacturer and model: Pixel 6 Pro
 - Android version: 13
 - App version: 1.17.0
 - App source (F-Droid, Google Play, Github): F-Droid

**Additional context**
N/A

**Comment by mateusz-bak** [comment]
Hi @hoshsadiq Thanks for bringing to my attention! It should be resolved in 1.17.1 that I released yesterday. You can wait couple of days for F-Droid release or take a cloud backup of your books and install .apk from Github!

**Comment by hoshsadiq** [comment]
Aah perfect! That works. Thank you
